source/jormungandr/tests/journey_common_tests.py

```diff
@@ -1612,11 +1612,17 @@ def test_one_region_without_pt_new_default(self):
         assert response['debug']['regions_called'][1] == "main_routing_test"
 
 
-@dataset({"main_routing_test": {'max_nb_journeys': 1}})
+@dataset(
+    {
+        "main_routing_without_pt_test": {"priority": 42, "min_nb_journeys": 10},
+        "main_routing_test": {"max_nb_journeys": 1},
+    }
+)
```

Comment:
If i understand, you added a new dataset to show that even with the lower priority value (0), "main_routing_test" is chosen as it has a solution with PT. As the name indicates main_routing_without_pt_test doesn't have any solution with PT ?

Reply:
You understood correctly. The dataset with the higher priority will be chosen first and its parameters set.
However, as there's no PT, the request will be done on the second dataset and its parameters will also be set: the result will only 1 journey as the parameter "max_nb_journeys = 1" is set for this coverage only